### PR TITLE
Fix the bug.

### PR DESCRIPTION
**Pull request by horsson**

### CODE REVIEW

File: HelloWorld/HelloWorld/ViewController.swift [hunk 1]
```diff
@@ -23,7 +23,7 @@ class ViewController: UIViewController {
 
 
     @IBAction func buttonDidClick(_ sender: Any) {
-        helloLabel.text = "Super Hello, world. TEST!"
+        helloLabel.text = "Super Hello, world."
```
Comment (haohucn) [inline HelloWorld/HelloWorld/ViewController.swift:26]: Why do you change it?

**Review by haohucn — CHANGES_REQUESTED** [review]
Please keep the old implementation.

File: HelloWorld/HelloWorld/ViewController.swift (hunk 1 above)
Comment (horsson) [reply HelloWorld/HelloWorld/ViewController.swift:26]: Because it's the requirement from my manager. She thinks, TEST here is kinda redundancy.

Comment (haohucn) [reply HelloWorld/HelloWorld/ViewController.swift:26]: Hi,
I think you need to follow our issue that is created by the PO.
Please revert it back.Thanks!

**Review by horsson — COMMENTED** [review]
Ok. I'll revert it.

**Review by haohucn — APPROVED** [review]
good approve.

**Pull request merged**